services/ingestion/etl/agri/csv/src/lib/transform.js

```diff
@@ -136,11 +137,8 @@ export default (record: Object): Project => {
     success_story: '',
     themes: [],
     timeframe: {
-      from:
-        timeframeFrom &&
-        new Date(parseInt(timeframeFrom, 10) * 1000).toISOString(),
-      to:
-        timeframeTo && new Date(parseInt(timeframeTo, 10) * 1000).toISOString(),
+      from: timeframeFrom && moment.unix(timeframeFrom).toISOString(),
```

Comment:
Dumb question: how is `moment.unix(timeframeFrom)` different from `new Date(parseInt(timeframeFrom, 10) * 1000)`? Is it worth including the whole `moment` library?

Reply:
By the way, if `timeframeFrom` is falsy, then `from` will be `undefined`. We only accept `string | null` ;)

Reply:
Thanks for noticing the need for more checks of falsy values, I will update for this.
`moment.unix` and other `moment` methods are more readable.